exporter/exporterhelper/queued_retry_experimental.go

```diff
@@ -62,6 +62,23 @@ func DefaultQueueSettings() QueueSettings {
 	}
 }
 
+// Validate checks if the QueueSettings configuration is valid
+func (qCfg *QueueSettings) Validate() error {
+	if !qCfg.Enabled {
+		return nil
+	}
+
+	if qCfg.QueueSize < 0 {
+		return fmt.Errorf("queue size can't be negative")
+	}
+
+	if qCfg.QueueSize == 0 {
+		return fmt.Errorf("a 0 size queue will drop all the data")
+	}
```

Comment:
Should this just be:

```golang
	if qCfg.QueueSize <= 0 {
		return fmt.Errorf("queue size must be positive")
	}
```

Reply:
Your suggestion makes a lot of sense. I originally went this path for consistency from looking at how we handle [queueSize in the Prometheus remote write exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/270b2c4cc10fb5188c0c839838a13781d9bd4ae4#diff-d000304209aed04ad612eb7941a129875e9244fd600c25c60ea8116c870f3e0a).

Curious on your thoughts here, should it be kept for consistency? (Also, apologies on the all the updates! I haven't used git rebase in a while. :) )

Reply:
Since the idea is to change the error on every exporter (including the PRW exporter), I think we can use the phrasing Bogdan suggested